### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by simrin051**
Files To Check

1. Cart List page (/pages/cart/CartList.jsx)
2. Product State Reducer  (/reducers/productStateReducer.jsx)
3. Product Service (/services/products.jsx)

### CODE REVIEW

File: src/pages/cart/CartList.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { useProductContext } from "../../contexts/products";
+import { CartItemCard } from './../../components/Card/CartItemCard';
+
+export const CartList = () => {
+    console.log("inside cart list");
```
Comment (zoyatanweer) [inline src/pages/cart/CartList.jsx:5]: remove console.log

**Review by zoyatanweer — APPROVED** [review]
code looks good to merge

File: src/pages/cart/CartList.jsx (hunk 1 above)
Comment (simrin051) [reply src/pages/cart/CartList.jsx:5]: Done

Comment (simrin051) [reply src/pages/cart/CartList.jsx:5]: Done

**Comment by simrin051** [comment]
@zoyatanweer  Thanks for reviewing my PR

**Pull request merged**